testkit/result.go

```diff
@@ -86,15 +86,15 @@ func Rows(args ...string) [][]interface{} {
 
 // Sort sorts and return the result.
 func (res *Result) Sort() *Result {
-	slices.SortFunc(res.rows, func(a, b []string) bool {
+	slices.SortFunc(res.rows, func(a, b []string) int {
 		for i := range a {
 			if a[i] < b[i] {
-				return true
+				return -1
 			} else if a[i] > b[i] {
-				return false
+				return 1
 			}
 		}
-		return false
+		return 1
```

Comment:
should we return 0?

Reply:
we need to add more logic like checking the length of slice.

Reply:
I have refactored it.